_posts/2017-05-10-april.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Dev Diary (April 2017)"
+author: "Corey Montella"
+tags: []
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://witheve.com))_
+
+In April we continued our effort to release v0.3 of Eve. On April 18th we released a second preview of the v0.3 platform, which included several long-requested features. However, toward the end of the month we shifted focus to revamping documentation and making the "getting started" experience for Eve more accessible to new users. Let's take a look at each of these efforts in this month's diary.
+
+### Platform
+
+We released a second preview of the v0.3 runtime, featuring multiple parent roots, which enable you to embed Eve-rendered elements into an application; and streamlined watchers with built in support for more DOM events and triggers in the HTML watcher. These are covered in detail in the release notes, but I'll take a moment now to highlight how multiple parent roots work.
```

Comment:
multiple parent roots isn't a useful idea to me yet, it's just implementation detail, better to tell people about what they've gained than how they've gained it.